internal/cli/config/rename.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/mongodb/mongocli/internal/prompt"
+	"github.com/spf13/cobra"
+)
+
+type RenameOpts struct {
+	oldName string
+	newName string
+}
+
+func (opts *RenameOpts) Run() error {
+	if config.Exists(opts.oldName) {
+		return fmt.Errorf("profile %v does not exist", opts.oldName)
+	}
+
+	if !config.Exists(opts.newName) {
```

Comment:
[q] should this be 
```suggestion
	if config.Exists(opts.newName) {
```
if it exists confirm?

Reply:
Oops yes updating this now